src/tulip/infos.ts

```diff
@@ -401,8 +705,33 @@ export class VaultInfoWrapper implements IVaultInfoWrapper {
   constructor(public vaultInfo: types.VaultInfo) {}
 
   getApr() {
-    // TODO
-    return 0;
+    const numberOfPeriods = 8760;
```

Comment:
Wonder how does `8760` come from? Perhaps add some descriptive comment here?

Reply:
Comment added.
this is yearly compound times = 24 * 365 = 8760

Reply:
Noted